### PR TITLE
[Relay][TOPI] Add rsqrt operator

### PR DESCRIPTION
**Pull request by makihiro**
This PR tries to add rsqrt operator to Relay with regard to https://github.com/dmlc/tvm/issues/2769
@GalAster @jroesch @ajtulloch @kazum please help to review.

### CODE REVIEW

File: python/tvm/hybrid/runtime.py [hunk 1]
```diff
@@ -36,6 +36,22 @@ def allocate(shape, dtype='float32', scope='global'): #pylint: disable=unused-ar
     return numpy.zeros(shape).astype(dtype)
 
 
+def rsqrt(x):
+    """
+    Computes reciprocal of square root of x element-wise
+
+    Parameters
+    ----------
+    x: a real number
```
Comment (jroesch) [inline python/tvm/hybrid/runtime.py]: Is this actually a real number, and not a tensor?

File: python/tvm/hybrid/runtime.py [hunk 2]
```diff
@@ -36,6 +36,22 @@ def allocate(shape, dtype='float32', scope='global'): #pylint: disable=unused-ar
...
+
+    Returns
+    -------
+    res: a real number
```
Comment (jroesch) [inline python/tvm/hybrid/runtime.py]: Same as above

File: python/tvm/relay/op/tensor.py [hunk 3]
```diff
@@ -63,6 +63,22 @@ def sqrt(data):
     return _make.sqrt(data)
 
 
+def rsqrt(data):
+    """Compute elementwise rsqrt of data.
```
Comment (jroesch) [inline python/tvm/relay/op/tensor.py]: Could we add a mathematical description of the op which will be rendered in the docs?

File: python/tvm/hybrid/runtime.py (hunk 1 above)
Comment (makihiro) [reply python/tvm/hybrid/runtime.py]: This is tensor, thanks.

File: python/tvm/hybrid/runtime.py (hunk 2 above)
Comment (makihiro) [reply python/tvm/hybrid/runtime.py]: This is tensor too, thanks.

File: python/tvm/relay/op/tensor.py (hunk 3 above)
Comment (makihiro) [reply python/tvm/relay/op/tensor.py]: Sure

File: python/tvm/relay/op/tensor.py [hunk 4]
```diff
@@ -63,6 +63,25 @@ def sqrt(data):
...
+
+    .. math::
+      1/sqrt(x)
```
Comment (tqchen) [inline python/tvm/relay/op/tensor.py]: rst requires a new blank line between .. math:: and the formula

Comment (makihiro) [reply python/tvm/relay/op/tensor.py]: Thank you. I fixed.

**Comment by tqchen** [comment]
Wait for @jroesch to approve and merge

**Comment by tqchen** [comment]
Thanks, @makihiro @jroesch this is now merged

**Pull request merged**